src/modules/leaderboard.js

```diff
@@ -1,9 +1,19 @@
+import profile1 from '../images/profile1.png';
+import profile2 from '../images/profile2.png';
+import profile3 from '../images/profile3.png';
+import profile4 from '../images/profile4.png';
+
 // Get relevant elements from the DOM
 const scores = document.getElementById('scores');
+const loader = document.getElementById('skeleton-loader');
+const empty = document.getElementById('empty-id');
 
 // URL to be used in fetch requests
 const URL = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${process.env.API_KEY}/scores/`;
```

Comment:
- [ ] **[OPTIONAL]** You are using all the data charged on the API and it is difficult to see my score on the list, please 🙏🏽  assign a valid name to the game for a better user experience. 👇🏽 

**HINT**
```
export const gameName = async (gameID) => {
  let response = await fetch(apiURL, {
    method: 'POST',
    body: JSON.stringify({ name: 'Ready player one' }),
    headers: {
      'Content-type': 'application/json; charset=UTF-8',
    },
  });
  response = await response.json();
  gameID = await response.result.slice(14, 34);
  localStorage.setItem('scores', gameID);
  window.location.reload();
};
```